rero_mef/ext.py

```
@@ -58,3 +60,57 @@ def register_signals(self, app):
+            except KeyError:
+                app.logger.info(
+                    "Skipping %s alias setup: search client is not available yet.",
+                    alias_name,
+                )
+                return
+
+            for target in target_aliases:
+                try:
+                    # Resolve alias -> concrete indexes.
+                    alias_mapping = search_client.indices.get_alias(name=target)
+                except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)